src/components/Comments/WriteComment.js

```diff
@@ -168,17 +169,17 @@ export const styles = StyleSheet.create({
     marginTop: 7,
     lineHeight: 15,
     fontSize: 12,
-    color: COLORS.gray8
+    color: colors.gray
   }),
   replyToTitle: {
     fontFamily: fonts.inter[600],
     lineHeight: 15,
     fontSize: 12,
-    color: COLORS.black
+    color: colors.black
   },
   container: (inReplyCommentView) => ({
     flex: 1,
-    backgroundColor: COLORS.white,
+    backgroundColor: '#fff',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of `replyToTitle` and the background color of `container` have been updated. Ensure these color changes align with your design guidelines.
```diff
- color: COLORS.gray8
- color: COLORS.black
- backgroundColor: COLORS.white,
+ color: colors.gray
+ color: colors.black
+ backgroundColor: '#fff',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->